packages/zero-client/src/client/context.test.ts

```diff
@@ -66,26 +71,30 @@ test('getSource', () => {
   expect(context.getSource('nonexistent')).toBeUndefined();
 
   // Should work for other table too.
-  const source2 = context.getSource('userStates');
+  const source2 = context.getSource('user_states');
```

Comment:
Looking at https://github.com/rocicorp/mono/pull/3684/files#diff-5add7e0d6b7b33e13c3a57846ae287c60e56e40352b80ad5ad26f64c870c2e26R338 it looks like the client still gets to refer to everything by client names but I don't see this test updated in that pr to go back to using `userStates`.